std_msgs/Header header
int32 class_id
string class_name
float32 confidence
float32 distance
float32[4] bbox
geometry_msgs/Point centroid_base
sensor_msgs/PointCloud2 in_box_cloud